jdbc/src/main/java/org/apache/zeppelin/jdbc/JDBCInterpreter.java

```diff
@@ -725,9 +611,17 @@ private InterpreterResult executeSql(String propertyKey, String sql,
                   "Query executed successfully.");
             } else {
               MutableBoolean isComplete = new MutableBoolean(true);
-              String results = getResults(resultSet,
-                  !containsIgnoreCase(sqlToExecute, EXPLAIN_PREDICATE), isComplete);
-              interpreterResult.add(results);
+
+              String results = getResults(resultSet, isComplete, getMaxResult());
+              List<ColumnDef.TYPE> columnTypes = getResultColumnTypes(resultSet);
+
+              final Boolean isTable = !containsIgnoreCase(sqlToExecute, EXPLAIN_PREDICATE);
```

Comment:
It is fragile to use `containsIgnoreCase` to check whether the result is table.  The following code can break it.

e.g.
```
select * from table where a = '  Explain '
```

Reply:
This remark does not apply to my changes. This line was already in zeppelin.